catalog/ftp/src/main/java/ddf/catalog/ftp/UserManagerImpl.java

```diff
@@ -197,7 +197,7 @@ private void setAdmin(User user) {
     if (StringUtils.isEmpty(this.admin)) {
       Subject subject = ((FtpUser) user).getSubject();
       for (String role : karafLocalRoles.split(",")) {
-        if (role.equalsIgnoreCase("admin") && subject.hasRole(role)) {
+        if ("admin".equalsIgnoreCase(role) && subject.hasRole(role)) {
```

Comment:
✏️ This is unnecessary; the elements of the array that results from calling `String.split` are never `null`.